sci-geosciences/GeographicLib/GeographicLib-1.51-r1.ebuild

```diff
@@ -0,0 +1,197 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+inherit cmake distutils-r1
+
+MY_PN="${PN,,}"
+
+DESCRIPTION="C++ library for converting geographic coordinate systems"
+HOMEPAGE="https://sourceforge.net/projects/geographiclib/"
+
+# To-Do:find out why this doesn't work
+# 105/167 Test #105: GeodSolve82 ......................***Failed
+# Required regular expression not found. Regex=[20\.0010* 0\.0* 180\.0* 20\.0010* 0\.0* 180\.0* 0\.0* 0\.0* 0\.0* 1\.0* 1\.0* -?0
+RESTRICT="test"
+
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE_GEOIDS_DATASETS="
+	geoids-datasets-egm84-30
+	geoids-datasets-egm84-15
+	geoids-datasets-egm96-15
+	geoids-datasets-egm96-5
+	geoids-datasets-egm2008-5
+	geoids-datasets-egm2008-2-5
+	geoids-datasets-egm2008-1
+"
+IUSE_GRAVITY_MODELS="
+	gravity-models-egm84
+	gravity-models-egm96
+	gravity-models-egm2008
+	gravity-models-wgs84
+"
+IUSE_MAGNETIC_MODELS="
+	magnetic-models-wmm2010
+	magnetic-models-wmm2015v2
+	magnetic-models-wmm2020
+	magnetic-models-igrf11
+	magnetic-models-igrf12
+	magnetic-models-emm2010
+	magnetic-models-emm2015
+	magnetic-models-emm2017
+"
```

Comment:
do we really need a useflag for every single one of those ?
it seems simpler to have a single useflag for all models

plus they don't seem to change anything on the installed programs, so maybe even worth a separate ebuild with a P/RDEPEND from geographiclib since they seem completely independent